libcontainer/utils/utils.go

```diff
@@ -144,17 +144,14 @@ func WithProcfd(root, unsafePath string, fn func(procfd string) error) error {
 
 // SearchLabels searches a list of key-value pairs for the provided key and
 // returns the corresponding value. The pairs must be separated with '='.
```

Comment:
Can you update the docs here to describe the purpose of the bool that's returned?